english/shotgun/shotgun.md

```diff
@@ -0,0 +1 @@
+You cock the shotgun and blast the polar bear in the face. Brain matter flys all over the marshmallow wall. Suddenly the brain matter starts to disintegrate the wall. You see sunlight peeking through. After a while the wall is gone and you are free. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.